packages/api/src/index.ts

```diff
@@ -2,11 +2,12 @@ import useInfiniteQuery from './useInfiniteQuery';
 import useMutation from './useMutation';
 import useQuery from './useQuery';
 
+export * as entity from './hooks/p2p/entity';
```

Comment:
This will be used in P2P like:
```
import { entity } from '@deriv/api'

const { data: paymentMethodsList } = entity.payment_method_entity.p2p_payment_methods.useGet()

```